build.gradle.kts

```diff
@@ -9,7 +9,9 @@ import java.util.Scanner
 val isUnstable = true
 version = "dev" + Scanner(Runtime.getRuntime().exec("git rev-list --count HEAD").inputStream).next() +
 		"-" + Scanner(Runtime.getRuntime().exec("git rev-parse --short HEAD").inputStream).next()
+var discordapi = "452444322004992001";
 file("src/resources/version").writeText(version as String)
+file("src/resources/discordapi").writeText(discordapi)
```

Comment:
as long as the key doesnt change frequently you dont need that, since the key is tracked anyways